src/Pinto/Gen.purs

```diff
@@ -35,11 +39,16 @@ foreign import init :: forall a. a -> a
 foreign import terminate :: forall a. a -> a -> a
 foreign import start_from_spec :: forall a. a -> a
 
+nativeName :: forall state msg name. ServerName state msg -> Foreign
+nativeName (Local name) = unsafeCoerce $ (atom name)
+nativeName (Global name) = unsafeCoerce $ tuple2 (atom "global") (atom name)
```

Comment:
Joined the church of `unsafeCoerce`!

If you wanted to be cheeky this could be `unsafeCoerce $ Global name` but yeah this

Reply:
hah - of course, I didn't even think of that - that's so bloody naughty.